bitnami/keycloak/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: keycloak
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/keycloak
   - https://github.com/keycloak/keycloak
-version: 12.1.5
+version: 12.1.6
```

Comment:
Thanks for the contribution!

This is adding a feature so the version should bump the minor (`12.2.0`)

Reply:
I changed the version to 12.2.0. Thank you for reviewing.